types/ssh2/index.d.ts

```diff
@@ -36,7 +36,34 @@ import {
 export import SFTP_STATUS_CODE = SFTPStream.STATUS_CODE;
 export import SFTP_OPEN_MODE = SFTPStream.OPEN_MODE;
 
-export { utils };
+export namespace utils {
```

Comment:
Please use exports from `ssh2-streams` instead of adding the methods and constants manually. I also believe it would be more correct to type this as an interface instead of a namespace.

Reply:
`utils` is already typed as a namespace in `@types/ssh2-streams`:

<https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1fe1620f70984cbe0c19bd4cad6bed14db7c26bf/types/ssh2-streams/index.d.ts#L1707-L1709>

And the exported `utils` is different in ssh2 compared to ssh2-streams

ssh2: <https://github.com/mscdex/ssh2/blob/2036f68de72601f56e21297b15886367c7457a36/lib/index.js#L34-L41>

ssh2-streams: <https://github.com/mscdex/ssh2-streams/blob/8de25ea69a76dbf1c34e578e42571e97de926ac0/lib/utils.js#L10-L18>

Reply:
Fair enough. Mind changing either `SFTPStream.OPEN_MODE` to `SFTP_OPEN_MODE` or `SFTP_STATUS_CODE` to `SFTPStream.STATUS_CODE` and adding semi-colons?

Reply:
Just pushed a commit to fix that